config/database.yml

```diff
@@ -28,19 +28,19 @@ development:
   # To create additional roles in PostgreSQL see `$ createuser --help`.
   # When left blank, PostgreSQL will use the default role. This is
   # the same name as the operating system user running Rails.
-  #username: chat_app
+  username: postgres
```

Comment:
Credential'ları hiç bir zaman hard coded olarak yazmamalıyız. Bunun yerine ENV variable'den çekeceklerimizi ENV'den credentials'da tutacaklarımızı ise rails.credentials'da tutalım. Bu bilgiler ENV specific de değişecek veriler bu sebeple ENV specific set edilmemiler. Bunu sağlayabileceğimiz bir yer env variable bir diğeri environment initializer bir diğer ise rails credentials'ın env set edilmiş hali. DB bağlantısı için en yaygın kullanım env varaibledir.